src/overlays/actors/ovl_En_Zl4/z_en_zl4.c

```diff
@@ -48,61 +61,1208 @@ const ActorInit En_Zl4_InitVars = {
+    /* 18 */ /* neutral, looking at Link */{ &gChildZeldaCYAnim_01805C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 19 */  /* moves towards link, hands clasped */{ &gChildZeldaCYAnim_018898, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
+    /* 20 */ /* facing link, hands clasped */ { &gChildZeldaCYAnim_01910C, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
+    /* 21 */ /* look in window */ { &gChildZeldaCYAnim_019600, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 22 */ /* leans forward, hands together */ { &gChildZeldaCYAnim_01991C, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, -1.0f },
+    /* 23 */ /* turns to link, hands on top of each other */ { &gChildZeldaCYAnim_01A2FC, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 24 */ /* stands, hands on top of each other */ { &gChildZeldaCYAnim_01AAE0, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 25 */ /* leaning forward, hands together */ { &gChildZeldaCYAnim_01AE88, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
+    /* 26 */ /* walks aside, points to window */ { &gChildZeldaCYAnim_01B874, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, 0.0f },
+    /* 27 */ /* stands pointing at window */ { &gChildZeldaCYAnim_01BCF0, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 28 */ /* laughs, hands together */ { &gChildZeldaCYAnim_01C494, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
+    /* 29 */ /* happy, hands together */ { &gChildZeldaCYAnim_01C7B0, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -1.0f },
+    /* 30 */ /* standing hands behind back looking down*/ { &gChildZeldaCYAnim_01CE08, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 31 */ /* cocks head, hands clasped */ { &gChildZeldaCYAnim_00F0A4, 1.0f, 0.0f, -1.0f, ANIMMODE_ONCE, 0.0f },
+    /* 32 */ /* happy, hands clasped */ { &gChildZeldaCYAnim_00F894, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    /* 33 */ /* transition to standing */ { &gChildZeldaCYAnim_000654, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, -8.0f },
+};
+
+#include "z_en_zl4_cutscene_data.c"
+
+void EnZl4_SetCsCameraAngle(GlobalContext* globalCtx, s16 index) {
+    Camera* activeCam = ACTIVE_CAM;
+
+    Camera_ChangeSetting(activeCam, 0x21);
+    activeCam->at = sCsCameraAngle[index].at;
+    activeCam->eye = activeCam->eyeNext = sCsCameraAngle[index].eye;
+    activeCam->roll = sCsCameraAngle[index].roll;
+    activeCam->fov = sCsCameraAngle[index].fov;
+}
+
+void EnZl4_SetCsCameraMove(GlobalContext* globalCtx, s16 index) {
+    Camera* activeCam = ACTIVE_CAM;
+    Player* player = PLAYER;
+
+    Camera_ChangeSetting(activeCam, 0x25);
+    Camera_ResetAnim(activeCam);
+    Camera_SetCSParams(activeCam, sCsCameraMove[index].atPoints, sCsCameraMove[index].eyePoints, player,
+                       sCsCameraMove[index].relativeToPlayer);
+}
+
+u16 EnZl4_GetText(GlobalContext* globalCtx, Actor* thisx) {
+    u16 faceReaction = Text_GetFaceReaction(globalCtx, 22);
+    u16 itemCount;
+    s16 ret;
+
+    if (faceReaction != 0) {
+        return faceReaction;
+    }
+
+    itemCount = 0;
```

Comment:
would prob call it `stoneCount` specifically or something, for the spiritual stones